### DISCUSSION TITLE
Strelka Backend Error

### DISCUSSION THREAD

**Discussion by xaqpuyrthnbg** [Security-Onion-Solutions/securityonion#2372] [General] [answered]
I noticed a couple days ago that Strelka Backend ends up in an error state after restart of the service, restart of the system, and state.apply.  I have not been able to get it stable again.  Any thoughts?

Thanks,

**Comment by gcsrellis** [comment, marked as answer]
Backend shuts down every 15 minutes if it isn't doing anything.  It's not a true error, it just says that when you do so-status.  Salt update restarts it on its own.  Unless you are having a true problem, this is a known "issue".

**Comment by xaqpuyrthnbg** [comment]
Thanks.  I ran so-status and I guess I just hadn't checked while it was failed.
